crypto/dalek/src/lib.rs

```diff
@@ -5,11 +5,11 @@
 
 #![no_std]
 
-/// Re-export of ed25519_dalek
-pub use ed25519_dalek::{self as ed25519};
-
 /// Re-export of curve25519_dalek
 pub use curve25519_dalek::{self as curve25519};
 
+/// Re-export of ed25519_dalek
+pub use ed25519_dalek::{self as ed25519};
+
 /// Re-export of x25519_dalek
 pub use x25519_dalek::{self as x25519};
```

Comment:
Optional/Nit: No need for `::self`
```suggestion
pub use curve25519_dalek as curve25519;

/// Re-export of ed25519_dalek
pub use ed25519_dalek as ed25519;

/// Re-export of x25519_dalek
pub use x25519_dalek as x25519;
```